.gitignore

```diff
@@ -56,4 +56,5 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
\ No newline at end of file
+.env
+.env.example

```

Comment:
@lawsmith I think we should probably continue to keep this in commits, since we're using this as the default for different environment variables right?